File: HackerHank/MigratoryBirds/MigratoryBirdsSolution.py
#
# Complete the 'migratoryBirds' function below.
#
# The function is expected to return an INTEGER.
# The function accepts INTEGER_ARRAY arr as parameter.
#

def migratoryBirds(arr):
    freq = [0,0,0,0,0,0]
    for i in range (0,len(arr)):
        freq[arr[i]]+=1
    maxi =0
    index =0
    for i in range (0, len(freq)):
        if(maxi<freq[i]):
            maxi=freq[i]
            index=i
    return index

File: HackerHank/MigratoryBirds/MigratoryBirdsSolution.cpp
/*
 * Complete the 'migratoryBirds' function below.
 *
 * The function is expected to return an INTEGER.
 * The function accepts INTEGER_ARRAY arr as parameter.
 */
#include <vector>
using namespace std;
int migratoryBirds(vector<int> arr)
{
    vector<int> frequency;
    for (int i = 0; i < 6; i++)
        frequency.push_back(0);
    for (int i = 0; i < arr.size(); i++)
        frequency[arr[i]]++;
    int max = 0;
    int index = 0;
    for (int i = 0; i < frequency.size(); i++)
        if (frequency[i] > max)
        {
            max = frequency[i];
            index = i;
        }
    return index;
}